src/bot.js

```diff
@@ -11,7 +11,11 @@ const { data } = require("./jsons"),
 function onMessage(bot, msg) {
   if (!msg.text) return; // no text
   const text = msg.text.toString();
-  if (text[0] !== "/") return; // no command
+  if (text[0] !== "/") {
```

Comment:
Mi ispira un sacco l'idea di rispondere agli spammer in automatico anziché dover aspettare che qualcuno lanci il comando. Unica cosa evitiamo di hardcodare il nome di un comando specifico in mezzo al codice come caso particolare. Se vuoi puoi aggiungere un file automemes.json e fare in modo che il bot controlli quelli. Se non ne hai voglia togli le modifiche a bot.js, ti approvo quelle a memes.json e apri un issue chiedendo a qualcun altro di farlo al posto tuo

Reply:
(Aggiornamento: ci sta lavorando @bugboy-20, al quale è stata assegnata questa PR)